swagger/swagger-invocation/invocation-core/src/main/java/org/apache/servicecomb/swagger/engine/SwaggerConsumer.java

```diff
@@ -34,14 +35,14 @@ public void setConsumerIntf(Class<?> consumerIntf) {
   }
 
   public void addOperation(SwaggerConsumerOperation op) {
-    operations.put(op.getSchemaOperationId(), op);
+    operations.put(op.getConsumerMethod(), op);
   }
 
-  public SwaggerConsumerOperation findOperation(String consumerMethodName) {
-    return operations.get(consumerMethodName);
+  public SwaggerConsumerOperation findOperation(Method consumerMethod) {
```

Comment:
can still keep `findOperation(String consumerMethodName)`, and mark as VisiableForTest  
this makes test easier

Reply:
Done. Please review it.